### DISCUSSION TITLE
Completely portable setting

### DISCUSSION THREAD

**Discussion by CyberHawk44** [Devolutions/UniGetUI#2807] [General]
Hello,
Is there already a possibility UniGetUI cpl. To make portable?
By that I mean that the settings, downloaded programs etc. are all in the UniGetUI folder.
If this option does not yet exist, I would like this.
It would also be helpful to incorporate something that you could put on an autostart afterwards.
Thank you very much for your work. :)
PS:
Maybe there is a way to give you one or more coffees or beer. ;)

PSS: I didn't know where to place it exactly, I just did it under ideas.

**Comment by marticliment** [comment]
Please open a "Feature Request" issue
